docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -279,9 +279,9 @@ testcafe --list-browsers
 
 ### -r \<name\[:output\],\[...\]\>, --reporter \<name\[:output\],\[...\]\>
 
-Specifies the name of a [built-in](../guides/concepts/reporters.md) or [custom reporter](../extending-testcafe/reporter-plugin/README.md) that is used to generate test reports.
+Specifies the name of a [built-in](../guides/concepts/reporters.md) or [custom reporter](../extending-testcafe/reporter-plugin/README.md) that generates test reports.
 
-The following command runs tests in all available browsers and generates a report in xunit format:
+The following command runs tests in all available browsers and generates a report in `xunit` format:
```

Comment:
Not related to this commit:

Boris recently said that `xunit` isn't a valid **format** name, it's `JUnit XML` (check this with Boris in case I'm mistaken).

`xunit` is just our reporter's name.

Reply:
I believe the format itself is called `XUnit XML`. 
The tool it was created for is called JUnit, but `XUnit` is a valid format name, I think
[Source](https://google.github.io/rich-test-results/xunitxml#logo:~:text=The%20XUnit%20XML%20format,-was)

Reply:
Okay, but isn't it better to just check it with Boris?

Reply:
Wikipedia says it's the collective name for several unit testing frameworks.
I believe xunit is a xml subset of xunit.net and junit-like formats. In our documentation context xunit is a correct name since it has its own specs, so @DIRECTcut is right.